static/app/components/contextPickerModal.spec.tsx

```diff
@@ -263,7 +264,7 @@ describe('ContextPickerModal', function () {
   it('is superUser and no integration configurations and calls `onFinish` with URL to integration overview page', async function () {
     OrganizationsStore.load([org]);
     OrganizationStore.onUpdate(org);
-    ConfigStore.config = TestStubs.User({isSuperuser: false});
```

Comment:
womp. Looks like this was just wrong before.

We probably aren't clearing the mock data between tests.